config/locales/client.en.yml

```
@@ -70,6 +70,11 @@ en:
         public_available: "There are public channels available for you to follow."
         start_direct_message: "You can also start a personal chat with one or more users."
         title: "Get started by joining a channel"
+      email_frequency:
+        description: "We'll only email you if we haven't seen you in the last 10 minutes."
```

Doesn't the job run every 15 minutes?

```suggestion
        description: "We'll only email you if we haven't seen you in the last 15 minutes."
```

---

This will be tied to the `user#last_seen_at` and the `email_time_window_mins` setting (default to 10 minutes). Maybe we could run the job every 10 minutes instead 15.